src/javascript/app_2/App/Components/Elements/DataTable/data-table.jsx

```diff
@@ -61,7 +67,8 @@ class DataTable extends React.PureComponent {
                         TableData
                         :
                         <Scrollbars
-                            style={{ width: '100%', height: 'calc(100vh - 35px)' }}
+                            style={{ width: '100%' }}
```

Comment:
Since you set it to `autoHeight` do you still need `style={{ width.. }}`?

Reply:
@zakibinary I can try it out without the width styling, but I'm not sure why `autoHeight` would affect `style={{ width }}`

Reply:
I don't believe `style={{ width }}` has any effect on `autoHeight`, which is why I asked whether it is still needed.